### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ List box _ Labels not announced 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3147]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	List box
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A) _ Modify Modules _ List box _ Labels not announced 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Activate ‘Edit’ tab from the main region.
•	Now navigate to ‘api-gateway’ , ‘btp-operator’ , ‘istio’ and ‘telemetry’ list boxes using tab key.

3. Issue Type: Manual
Observed Behavior:
On accessing ‘api-gateway’ , ‘btp-operator’ , ‘istio’ and ‘telemetry’ list boxes within ‘Edit’ tab of the page, the respective labels of the input fields i.e. ‘api-gateway’ , ‘btp-operator’ , ‘istio’ and ‘telemetry’ is not announced by the  screen reader. 
Screenshot:

 
 
Expected Behavior:
On activating ‘Add’/’Delete’ module a user can add/delete a particular module to the list and the success message ‘Module was added/deleted’ is displayed to the user, and  this success information i.e. ‘Module was added/deleted’ must be announced by the screen reader along with other necessary information. 
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-264.1 (Level A) _ Modify Modules _ List box _ Labels not announced.docx](https://github.com/user-attachments/files/16660074/ACC-264.1.Level.A._.Modify.Modules._.List.box._.Labels.not.announced.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660075/Issue.processing-09.2021.002.docx)
